google/cloud/storage/examples/storage_grpc_samples.cc

```diff
@@ -62,7 +62,7 @@ void GrpcReadWrite(std::string const& bucket_name) {
 void GrpcClientWithDP() {
   namespace g = ::google::cloud;
 
-  auto client = google::cloud::storage_experimental::DefaultGrpcClient(
+  auto client = google::cloud::storage::MakeGrpcClient(
       g::Options{}.set<g::EndpointOption>(
           "google-c2p:///storage.googleapis.com"));
```

Comment:
this is no longer required right?

Reply:
Good catch. No longer needed as now is the default if "detected".